docs/build/linux-ubuntu.md

```diff
@@ -191,3 +206,59 @@ older versions, you can set `JAVA_HOME_11` to the OpenJDK 11 path and
 from command line. But note that you do need to update your `JAVA_HOME` env
 var to the associated version before opening Android Studio and loading any
 Android projects. `JAVA_HOME_11` is mostly used by CI.
+
+## Docker Builder
```

Comment:
good, should we _remove_ the other discussion of this Dockerfile then?

Reply:
Which other discussion do you mean?